zeebe/engine/src/test/java/io/camunda/zeebe/engine/processing/job/JobUpdateTest.java

```diff
@@ -42,14 +43,16 @@ public void setup() {
   public void shouldUpdateJob() {
```

Comment:
🔧 We could add test cases for
* Conversion of the changed attribute value is failing (we accept any `Number` types currently, I think, we can probably make that fail in a test if we want to to see if our processor can handle that 🙂)
* Providing no changed attribute - what happens in our processor in this case? I know we validate this in the REST API but the processor is decoupled from that, it doesn't know that. So it needs to be prepared.
*

Reply:
I didn't add more tests cases yet, but thanks for the suggestions